Xamarin.Forms.Core/ListProxy.cs

```diff
@@ -436,6 +436,9 @@ public void Reset()
 
 		#region IList
 
+		bool IListProxy.TryGetValue(int index, out object value)
```

Comment:
if the explicit implementation forwards to the internal implementation, why not doing a single explicit implementation ?

you will still be able to invoke it, as the interface is public